Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Define base image.
-FROM nvidia/cudagl:11.3.1-devel
+FROM nvidia/cuda:11.7.1-devel-ubuntu22.04
```

Comment:
Will this still work with windows?

Reply:
Yes, also the old image was based on Ubuntu (version 20.04), you simply need the docker environment based on WSL and Nvidia drivers as before.
Actually, this shouldn't make any difference from the user perspective.

Reply:
Gotcha, (docker beginner here)